client/lib/cart/store/index.js

```diff
@@ -38,7 +34,6 @@ import {
 	fillInAllCartItemAttributes,
 	setTaxCountryCode,
 	setTaxPostalCode,
-	setTaxLocation,
```

Comment:
I don't think `setTaxLocation` is used anywhere else.

Reply:
I think we might as well leave this for now as it's part of the old CartStore which will be deleted wholesale at some point soon 🤞

Reply:
Nevermind, I removed it anyway.